### DISCUSSION TITLE
Error when using TabPanels

### DISCUSSION THREAD

**Discussion by nowiko** [chakra-ui/chakra-ui#6862] [Q&A]
Why I am getting the following error message:
`Uncaught ContextError: useTabsContext: `context` is undefined. Seems you forgot to wrap all tabs components within <Tabs />`

When trying to use code from this example - https://codesandbox.io/s/pedantic-pine-d1oui3?file=/package.json:123-351

I am using the same dependencies list,  could someone, please help me?

**Comment by tae100k** [comment]
Did you find any solutions?
